csrankings-y.csv

```diff
@@ -747,7 +747,7 @@ Yuan Tang,Fudan University,http://people.csail.mit.edu/yuantang,kC2YD14AAAAJ
 Yuan Tian 0001,University of Virginia,https://www.ytian.info,ja0GtqgAAAAJ
 Yuan Tian 0008,Queen’s University,http://sophiaytian.com,wX4le_QAAAAJ
 Yuan Wu 0001,University of Macau,https://www.fst.um.edu.mo/people/yuanwu/,H1bxY_4AAAAJ
-Yuan Xie 0001,Univ. of California - Santa Barbara,http://www.ece.ucsb.edu/~yuanxie,dK2ZuDcAAAAJ
+Yuan Xie 0001,HKUST,https://seng.hkust.edu.hk/about/people/faculty/yuan-xie,dK2ZuDcAAAAJ

```

Comment:
Not in CS; explain how they meet eligibility criteria.

Reply:
Showing on his faculty page (https://ece.hkust.edu.hk/yuanxie), Professor Yuan closely works in the following fields that appear in the categories of the CSRankings website:
* Computer architecture
* Design Automation (EDA)
* Machine Learning

Also, most of his publications focus on these three fields as listed on his Google Scholar page: https://scholar.google.com/citations?hl=en&user=dK2ZuDcAAAAJ

He is able to solely instruct computer science Ph.D. students. Here is a justification (the webpage shows one of his Ph.D. student in the CS department): https://www.ce.ucsb.edu/spotlights/liu-liu-phd-student-cs